### PR TITLE
fix(website): to differentiate between internal and external links

### PR DESCRIPTION
**Pull request by emmenko**
Inspired by https://www.gatsbyjs.org/docs/gatsby-link/#reminder-use-link-only-for-internal-links

### CODE REVIEW

File: website/src/templates/markdown.js [hunk 1]
```diff
@@ -216,18 +216,25 @@ const Link = props => {
...
+    // E.g. if `href="./foo"` and we're on page `/getting-started/bar`, the
+    // value we need is `/getting-started/foo`.
+    // To achieve that, we let the DOM API build the full URL, then we simply
+    // extract the relative path.
+    const linkElement = document.createElement('a');
+    linkElement.href = props.href;
+    const absoluteUrl = linkElement.href; // <-- this now is the full absolute URL
+    const [, relativePath] = absoluteUrl.split(window.location.origin);
+    setLink(relativePath);
+  });
```
Comment (tdeekens) [inline website/src/templates/markdown.js]: Don't have have to specify `[props.href]` as a dep? Should we add the ESLint rules for hooks on the pkg?

Comment (emmenko) [reply website/src/templates/markdown.js]: The prop won't change, so it shouldn't be necessary. Or am I missing something?

Comment (tdeekens) [reply website/src/templates/markdown.js]: In that case it's fine. It's just underspecifying the dependencies which I read is rarely what one wants and can lead to quite silent bugs.

Comment (emmenko) [reply website/src/templates/markdown.js]: Yeah I know. I can add an empty array if you want, to make it "safer"?

Comment (tdeekens) [reply website/src/templates/markdown.js]: I think that's the same. Anyway. It's fine just wanted to make sure.

File: website/src/templates/markdown.js [hunk 2]
```diff
@@ -192,22 +191,59 @@ const Link = styled.a`
...
+        css={css`
+          display: inline-flex;
+          align-items: center;
+        `}
+      >
+        <a
+          {...props}
+          css={linkStyles}
+          target="_blank"
+          rel="noopener noreferrer"
```
Comment (montezume) [inline website/src/templates/markdown.js]: 👍

**Pull request merged**